src/components/sources/OlSourceWMTS.vue

```diff
@@ -7,6 +7,7 @@ import type { RequestEncoding, Options } from "ol/source/WMTS";
 import WMTS from "ol/source/WMTS";
 import type { Options as ProjectionOptions } from "ol/proj/Projection";
 import Projection from "ol/proj/Projection";
+import TileGrid from "ol/tilegrid/TileGrid";
```

Comment:
```suggestion
```

Why no using the specific TileGrid for WMTS as described in the source vector docs: https://openlayers.org/en/latest/apidoc/module-ol_source_WMTS-WMTS.html